src/ontology/edits/oeo-model.omn

```diff
@@ -3811,6 +3811,19 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1945",
         <http://purl.obolibrary.org/obo/BFO_0000023>
     
     
+Class: OEO_00400041
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A model system boundary is a model component that separates aspects of the real world that are included in the model from those that are not.",
```

Comment:
I don't know how firmly the wording was agreed on but I think 

"...aspects of the real world included in the model" would sound better than using "that" twice in a row.

Reply:
Good point

Reply:
Alright, I fixed it